modules/oe_translation_cdt/src/Mapper/TranslationRequestMapper.php

```diff
@@ -173,11 +175,17 @@ protected function countVolume(int $character_count): float {
   protected function createCallbacks(): CallbackCollection {
     $job_status_url = Url::fromRoute(
       route_name: 'oe_translation_cdt.request_status_callback',
```

Comment:
Lets add a comment explaining why we did it

Reply:
done